PusherServer/BatchTriggerBody.cs

```diff
@@ -0,0 +1,11 @@
+﻿
+namespace PusherServer
+{
+    /// <summary>
+    /// Represents the payload to be sent when triggering events
+    /// </summary>
+    class BatchTriggerBody
+    {
+        public BatchEvent[] batch { get; set; }
```

Comment:
Maybe this isn't a public API so it may be fine. Although there are ways of controlling the name of the serialised property.